app/services/account_reset/grant_requests_and_send_emails.rb

```diff
@@ -7,12 +7,18 @@ class GrantRequestsAndSendEmails < ApplicationJob
     good_job_control_concurrency_with(
       enqueue_limit: 1,
       perform_limit: 1,
-      key: -> { "grant-requests-and-send-emails-#{arguments.first}" },
+      key: -> do
+        now = arguments.first
+        five_minutes = 5.minutes.to_i
+        rounded = (now.to_i / five_minutes) * five_minutes
+
+        "grant-requests-and-send-emails-#{rounded}"
+      end
     )
 
     discard_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
 
-    def perform(_date)
+    def perform(_now)
```

Comment:
we use `Time.zone.now` inside the job.... but due to the way the tests were written, it was not easy to just drop that in, future work could be to clean that up